doc/user_quide.asciidoc

```diff
@@ -14,85 +14,74 @@ Latest Version: {Latest_Version}
 
-=== How does distribution migration system do it ?
-To achieve the migration between this two releases, the entire migration process
-is called from within a Live Migration Image.
+The migration between two major distribution versions requires the system to be migrated to be off line to avoid system inconsistencies that may leave the system in a state that does not allow recovery. This behavior is implemented using a Live Migration Image.
 
-Once the Distribution Migration system package has been installed in
-the system to be migrated and it starts running, a pipeline of systemd
-controlled services start to run. The following chain of services will be executed:
+The distribution migration system provides the live image and a component that modifies the bootloader configuration such that on the next boot the system will boot into the migration image. Once booted into the migration live image the following chain of services will be executed:
 
-- detect the system to migrate
-- mount it
-- setup the network as clone from the system to migrate
+- detect the system to be migrated
+- mount the necessary file systems
+- setup the network to match the network configuration of the system to be migrated
 - prepare the package manager for the migration task
 - migrate the system
-- update the grub
-- unmount what it mounted
+- update the boot loader configuration
+- unmount all mounted file systems
 - reboot
 
+Should an error occur prior to the start of the actual migration the system will be returned to it's original state.
+
+=== Requirements
+The system to be migrated must be registered. Pay as you go instances in the Public Cloud are automatically registered to the SUSE operated update infrastructure. All other systems must be connected to the SUSE Customer Center (SCC), a Subscription Management Tool (SMT), or a Repository Management tool (RMT) server. For system managed via SUSE Manager use the migration path provided by SUSE Manager. 
+
+
+
 === Installation
 
-Inside the system to be migrated the following tasks have to be performed:
+The distribution migration system is made available in the Public Cloud module. Therefore this module has to be enable on the system to be migrated. For running on-demand instances this module is already enabled in the instance.
 
-. fetch the Distribution Migration System package which contains a
-KIWI built image based on the target distribution of the migration:
-+
-[listing]
-$ zypper ar official_repo_url live_migration
+For data center customers it is recommended to continue to use the documented offline distribution migration using the next distribution version installation media.
 
-. fetch the suse-migration-activation package which contains an
-activation package:
-+
-[listing]
-$ zypper ar official_repo_url suse_migration_activation
+To install the distribution migration system execute as root
 
-. install it:
 +
 [listing]
 $ zypper in SLE15-Migration suse_migration_activation
```

Comment:
should we not point to the repos to find those packages?

Reply:
Let's discuss this on the main PR branch. I agree the docs should include the location from where to get the packages. This would be the SLE12 Public Cloud Module. However in the public cloud environment and in an on-demand instance those repos will already be registered through our SMT/RMT infrastructure. I also think this should be mentioned with regards to use in public clouds